globalsearchv2/global_search_v2.go

```diff
@@ -403,13 +431,13 @@ func UnmarshalResultItem(m map[string]json.RawMessage, result interface{}) (err
 // ScanResult : The search scan response.
 type ScanResult struct {
 	// The search cursor to use on all calls after the first one.
-	SearchCursor *string `json:"search_cursor" validate:"required"`
+	SearchCursor *string `json:"search_cursor,omitempty"`
 
 	// Value of the limit parameter specified by the user.
-	Limit *int64 `json:"limit,omitempty"`
+	Limit *float64 `json:"limit" validate:"required"`
```

Comment:
The "limit" property of the ScanResult schema should most likely be defined as "integer" instead of "number". Apparently, back when this service was last re-generated (by @damianovesperini on 6/10/2021) the property was in fact defined as "integer" which is why the current version of the code is using "int64".  It looks like search.yaml was updated in [this commit](https://github.ibm.com/cloud-api-docs/tagging/commit/a89b4779a1496f75528229764fe497459940f16b) to use "number" instead of "integer".  I think this should be changed back to "integer" in the API definition.

Reply:
At this question can replay Michele Crudele.

Reply:
It will be fixed

Reply:
@fabrizio-leoni When the API definition is fixed for this problem and the endpoint URL issue, you'll need to regen the SDK code one more time, then update the PR with the changes.  Also, be sure to pull in the latest changes from the main branch as well.